packages/atomic/src/utils/initialization-utils.tsx

```diff
@@ -54,6 +60,12 @@ export interface InitializableComponent extends ComponentInterface {
   error: Error;
 }
 
+export function applyFocusVisiblePolyfill(element: HTMLElement) {
+  if (window.applyFocusVisiblePolyfill && element.shadowRoot) {
```

Comment:
Why is `element.shadowRoot` part of the condition?

Reply:
Some components might not have shadow dom, in that case they don't need to set the event listeners a second time